File: algorithm/485. Max Consecutive Ones/485.py
class Solution(object):
    def findMaxConsecutiveOnes(self, nums):
        """
        :type nums: List[int]
        :rtype: int
        """
        tem,max=0,0
        for i in range(len(nums)):
            if nums[i]==0:
                if tem>max:
                    max=tem
                tem=0
            else:
                tem+=1
        if tem>max:
            max=tem
        return max

File: algorithm/485. Max Consecutive Ones/485.cpp
class Solution {
public:
    int findMaxConsecutiveOnes(vector<int>& nums) {
        int max=0,tem=0;
        for(int i=0;i<nums.size();i++)
        {
            if(nums[i]==0)
            {
                if(tem>max)
                    max=tem;
                tem=0;
            }
            else
                tem++;
        }
        if(tem>max)
            max=tem;
        return max;
    }
};
